types/types.go

```diff
@@ -26,7 +26,7 @@ type ComponentVersion struct {
 // Cluster type definition
 type Cluster struct {
 	ID         string             `json:"id"`
-	FirstSeen  time.Time          `json:"firstSeen"`
-	LastSeen   time.Time          `json:"lastSeen"`
+	FirstSeen  time.Time          `json:"firstSeen,omitempty"`
+	LastSeen   time.Time          `json:"lastSeen,omitempty"`
```

Comment:
Can you add some short documentation here to indicate if there's a semantic difference between empty and non-empty timestamps?